src/callbacks_step/analysis.jl

```diff
@@ -195,6 +201,14 @@ function (analysis_callback::AnalysisCallback)(integrator)
   @unpack dt, t = integrator
   iter = integrator.destats.naccept
 
+  # Record performance measurements and compute walltime/coretime PID
+  runtime_since_last_analysis = 1.0e-9 * (time_ns() - analysis_callback.start_time_last_analysis)
+  ncalls_rhs_since_last_analysis = (semi.performance_counter.ncalls_since_readout -
+                                    analysis_callback.ncalls_rhs_last_analysis)
+  ndofsglobal = div(ndofs(semi), nelements(mesh, solver, cache)) * nelementsglobal(mesh, solver, cache)
```

Comment:
This assumes that `ndofs` gives the local number of DOFs, `nelements` gives the local number rof elements, and that all elements have the same number of DOFs, correct? I would prefer having `ndofs_global` or so instead.

Reply:
Done in 56c3afe0.